src/pages/landing/jump-indices/_dHero.js

```diff
@@ -0,0 +1,175 @@
+import React, { createRef, useEffect } from 'react'
+import lottie from 'lottie-web'
```

Comment:
Should this be from 'react-lottie'?

Reply:
Cause there should be a `<Lottie />` component we can use instead of creating refs and running useEffect.

Reply:
I'm using `lottie-web` because its easier to handle styling for the animation as the designer wants to add masking on the left side of the animation..I've tried adding the styling on `react-lottie` but with no success.